### PR TITLE
Add 'condition' option to ol.interaction.DragPanOptions in olx.js

### PR DESCRIPTION
**Pull request by SamuelBerger**
As discribed in #5518

### CODE REVIEW

File: externs/olx.js [hunk 1]
```diff
@@ -2432,12 +2432,23 @@ olx.interaction.DragBoxOptions.prototype.boxEndCondition;
 
 
 /**
- * @typedef {{kinetic: (ol.Kinetic|undefined)}}
+ * @typedef {{condition: (ol.events.ConditionType|undefined),
```
Comment (fredj) [inline externs/olx.js]: must be `ol.EventsConditionType` instead of `ol.events.ConditionType`

Comment (SamuelBerger) [inline externs/olx.js]: ups... 
To keep a single commit, is a force push ok?

Comment (ahocevar) [inline externs/olx.js]: Yes, please force-push to your branch.

**Comment by fredj** [comment]
Thanks for th patch @SamuelBerger. Dit you already signed the [Contributor License Agreement](https://github.com/openlayers/ol3/blob/master/CONTRIBUTING.md#contributor-license-agreement) ?

**Comment by ahocevar** [comment]
ICLA received.

**Comment by SamuelBerger** [comment]
Thanks for merging!

**Pull request merged**